__tests__/read/1.html

```diff
@@ -1,3 +1,8 @@
+<ng-container *transloco="let t; read: 'ternary.nested'">
+  <p>{{t('test')}}</p>
+  <p>{{t(condition ? 'yin' : 'yang')}}</p>
```

Comment:
Please add more permutations and see that your RegExp is handling them correctly.
For example: `condition ? "yin" : 'yang'` won't work